backend/plugin/db/oracle/oracle.go

```diff
@@ -57,6 +57,11 @@ func (driver *Driver) Open(_ context.Context, _ db.Type, config db.ConnectionCon
 	if err != nil {
 		return nil, err
 	}
+	if config.SchemaTenantMode {
+		if _, err := db.ExecContext(ctx, fmt.Sprintf("ALTER SESSION SET CURRENT_SCHEMA = %s", driver.databaseName)); err != nil {
```

Comment:
Does Oracle support query parameter? Trying to find a way to avoid SQL injection and escaping.